CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(LLVMIR_OPT ${SEA_OPT})
 set(AWS_C_COMMON_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/aws-c-common)
 set(SEA_LIB ${CMAKE_CURRENT_SOURCE_DIR}/seahorn/lib)
 
-configure_file(verify.py.in verify @ONLY)
+configure_file(verify_in.py verify @ONLY)
```

Comment:
please change the name back. By convention, '*.in' files are the ones processed by build scripts.
At least my editor recognizes the convention and opens this file in python mode automatically.

Reply:
Without a py extension, I cannot import this file as a module using the standard method.
If I change it to run on the final `verify` script, it faces the same problem.

Reply:
One solution can be to change the final script to verify.py taking into account the review comment to move the test into the build dir and run it using cmake.

Reply:
t￼he source of the problem is that you are treating binary as a library because python lets you. 

dont. 

nuke the test. it is not needed